src/expr/src/scalar/func.rs

```diff
@@ -2115,6 +2115,7 @@ fn jsonb_pretty<'a>(a: Datum<'a>, temp_storage: &'a RowArena) -> Datum<'a> {
     Ord, PartialOrd, Clone, Debug, Eq, PartialEq, Serialize, Deserialize, Hash, MzEnumReflect,
 )]
 pub enum BinaryFunc {
+    NotImplemented(String),
```

Comment:
could you either add a comment explaining what the string is or change it to a self-documenting item?

```suggestion
    NotImplemented { not_implemented_item: String },
```